### ISSUE TITLE
Reformatting examples in clsguide.tex

### ISSUE THREAD

**Issue by dr-scsi** [latex3/latex2e#1282]
Hi,

the current `clsguide.tex` contains these 2 examples:
https://github.com/latex3/latex2e/blob/8809ac6fc68fa5caa8a94281ee6134880e6c1f11/base/doc/clsguide.tex#L1106-L1112
and
https://github.com/latex3/latex2e/blob/8809ac6fc68fa5caa8a94281ee6134880e6c1f11/base/doc/clsguide.tex#L1120-L1127

Maybe I'm old school, but would you consider rewriting them as:
```
\newenvironment{texteqn}{%
  \begin{equation}
  \begin{minipage}{0.9\linewidth}
  }{%
  \end{minipage}
  \end{equation}
}
```
and
```
\newenvironment{texteqn}{%
  \begin{equation}
  \begin{minipage}{0.9\linewidth}
  }{%
  \end{minipage}
  \end{equation}
  \ignorespacesafterend
}
```
or something similar?  I find the current indentation make the examples really hard to parse.  TIA.

**Comment by josephwright** [comment]
Checking the log, those lines are the same in `clsguide-historic`, and have been unchanged for at least 16 year :)

**Comment by dr-scsi** [comment]
It seems that I read that part carefully for the very first time :-)

**Comment by josephwright** [comment]
I was checking really as I wondered how the formatting came from: the log says Rainer but that could well mean he was doing admin for someone else. The style reminds me of Philipp Lehman's (`biblatex` original author).

**Comment by muzimuzhi** [comment]
Personally I prefer
```tex
\newenvironment{texteqn}{%
  \begin{equation}
    \begin{minipage}{0.9\linewidth}
}{%
    \end{minipage}
  \end{equation}
}
```
